chunky/src/java/se/llbit/chunky/renderer/RenderManager.java

```diff
@@ -21,8 +21,10 @@
 import se.llbit.log.Log;
 import se.llbit.util.TaskTracker;
 
+import java.awt.*;
```

Comment:
Is AWT actually used in this class?

Reply:
Whoops, I thought my IDE automatically cleaned imports.